examples/features/torch_batch_process/batch_inference/compare_with_core_api/README.md

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
no empty start line, please.

It's the one exception to the "empty line above and below each header" rule.